src/game.cpp

```diff
@@ -4066,38 +4066,46 @@ void Game::showPauseMenu()
 		<< "textarea[0.4,0.25;3.9,6.25;;" << PROJECT_NAME_C " " VERSION_STRING "\n"
 		<< "\n"
 		<<  strgettext("Game info:") << "\n";
-	const std::string &address = client->getAddressName();
+	const ServerInfo *server_info = client->getServerInfo();
+
+	// cache strings we use a lot
 	static const std::string mode = strgettext("- Mode: ");
+	static const std::string on = strgettext("On");
```

Comment:
translate strings can be better

Reply:
Would something like "enabled" and "disabled" work better for translators?

Reply:
Yes. Translation for "enabled" and "disabled" already exist.

Reply:
@srifqi so should "on" and "off" but I can see how that might be harder on translators due to having less context.

Reply:
Resolved?